components/edit-collective/Form.js

```diff
@@ -49,6 +49,7 @@ import PaymentReceipts from './sections/PaymentReceipts';
 import Policies from './sections/Policies';
 import ReceivingMoney from './sections/ReceivingMoney';
 import SendingMoney from './sections/SendingMoney';
+import ThankYouEmail from './sections/ThankYouEmail';
```

Comment:
This should be renamed `CustomEmail`

Reply:
I've renamed all the stings with `ThankYouEmail` to `CustomEmail` including the setting `collective.settings.thankYouEmailMessage` to `collective.settings.customEmailMessage` so that things are consistent.